src/main/webapp/scripts/controller-auth.js

```diff
@@ -0,0 +1,61 @@
+import { postVideoComment } from '/scripts/post-comment.js';
+import { videoId, commentElement, failedInitCallback, successfulApiCallback, failedApiCallback, updateSigninStatus } from '/scripts/view.js';
+
+/** API key provied by GCP to authenticate the client*/
+const apiKey = '';
+/** client ID provied by GCP to identify the client*/
+const clientId = '';
+/** discovery docs used to identify API endpoints */
+const discoveryDocs = ['https://www.googleapis.com/discovery/v1/apis/youtube/v3/rest'];
+/** OAuth2 scope required to post a comment */
+const scope = 'https://www.googleapis.com/auth/youtube.force-ssl';
+
+/** Load the client from Google API oauth2.0 JS library*/
+function handleClientLoad() {
+  gapi.load('client:auth2', initClient);
+}
+
+/** 
+ * Initialize the client with appropiate fields.
+ * And listen for changed in the signin status
+*/
+function initClient() {
+  gapi.client.init({
+    apiKey,
+    discoveryDocs,
+    clientId,
+    scope
+  })
+    .then(function () {
+      // Listen for sign-in state changes.
+      gapi.auth2.getAuthInstance().isSignedIn.listen(updateSigninStatus);
+
+      // Handle the initial sign-in state.
+      updateSigninStatus(gapi.auth2.getAuthInstance().isSignedIn.get(),
+        handleAuthClick, handleSignoutClick, handleCommentClick);
+    })
+    .catch(failedInitCallback);
+}
+
+/** 
+ * Handler that initiates the authorization flow with a popup.
+ * The user only needs to authorize the site once.
+*/
```

Comment:
nit: this should line up i.e. 

```
/** 
 * Handler that initiates the authorization flow with a popup.
 * The user only needs to authorize the site once.
 */
```

please also fix elsewhere

Reply:
Done thanks for pointing it out :D